test/integration/targets/docker_network/tasks/tests/basic.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Registering container name
+  set_fact:
+    cname_1: "{{ name_prefix ~ '-container-1' }}"
+    cname_2: "{{ name_prefix ~ '-container-2' }}"
+    cname_3: "{{ name_prefix ~ '-container-3' }}"
+    nname_1: "{{ name_prefix ~ '-network-1' }}"
+    nname_2: "{{ name_prefix ~ '-network-2' }}"
+- name: Registering container name
+  set_fact:
+    cnames: "{{ cnames }} + [cname_1, cname_2, cname_3]"
+    dnetworks: "{{ dnetworks }} + [nname_1, nname_2]"
+
+- name: Create containers
+  docker_container:
+    name: "{{ container_name }}"
+    image: hello-world
+    state: present
+  loop:
+  - "{{ cname_1 }}"
+  - "{{ cname_2 }}"
+  - "{{ cname_3 }}"
+  loop_control:
+    loop_var: container_name
+
```

Comment:
no idempotency check for container ?

Reply:
No, that's already done enough many times in the docker_container integration tests ;)